test/unit/transformers/moduleNotFound.spec.js

```diff
@@ -22,8 +23,7 @@ it('Sets the appropiate message', () => {
   expect(moduleNotFound(error).message).toEqual(message);
 });
 
-it('Sets the appropiate message', () => {
-  const message = 'Module not found redux';
+it('Sets the appropiate type', () => {
```

Comment:
This test actually tests the `type`, not the `message` and the `message` variables was unused. Just a little cleanup.